pkgs/tools/graphics/pngcrush/default.nix

```diff
@@ -18,7 +20,7 @@ stdenv.mkDerivation rec {
     homepage = http://pmt.sourceforge.net/pngcrush;
     description = "A PNG optimizer";
     license = stdenv.lib.licenses.free;
-    platforms = with stdenv.lib.platforms; linux;
+    platforms = with stdenv.lib.platforms; [ linux darwin ];
```

Comment:
```platforms.linux``` and ```platforms.darwin```` already are lists. You have to concatenate them via `++` instead of putting both lists in another list.